src/components/Avatar.tsx

```diff
@@ -49,6 +49,9 @@ type AvatarProps = {
 
     /** Owner of the avatar. If user, displayName. If workspace, policy name */
     name?: string;
+    // this prop is used in RoomHeaderAvatars
+    // eslint-disable-next-line react/no-unused-prop-types
+    id?: string | number;
```

Comment:
This change isn't necessary after looking into code, let's use a different type in RoomHeaderAvatars